### PR TITLE
Add pytorch-ignite.

### PR DESCRIPTION
**Pull request by knc6**
Checklist
- [ ] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [ ] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/main/recipes/example/meta.yaml#L57-L66) for an example).
- [ ] Source is from official source.
- [ ] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [ ] Build number is 0.
- [ ] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [ ] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

File: recipes/pytorch-ignite/LICENSE [hunk 1]
```diff
@@ -0,0 +1,29 @@
+BSD 3-Clause License
```
Comment (beenje) [inline recipes/pytorch-ignite/LICENSE:1]: The LICENSE is included in the sdist on pypi. No need to add it here.

File: recipes/pytorch-ignite/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/pytorch-ignite-{{ version }}.tar.gz
+  sha256: 434ebb7ee5e3da1ec00812e4280487d23fef8cb32c6790bdf2cb43d7f26a87e7
+
+build:
+  skip: true  # [win]
+  number: 0
+#   noarch: python
```
Comment (beenje) [inline recipes/pytorch-ignite/meta.yaml:16]: This is a pure Python package. It should be noarch even if tests fails on windows due to missing dependencies.
Noarch packages are only built on Linux in the end.

File: recipes/pytorch-ignite/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python
```
Comment (beenje) [inline recipes/pytorch-ignite/meta.yaml:22]: Add a minimum version for both host and run

File: recipes/pytorch-ignite/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+  run:
+    - python
+    - pytorch
```
Comment (beenje) [inline recipes/pytorch-ignite/meta.yaml:25]: The package has some min and max requirements that you should keep: `>=1.3,<2`

Comment (knc6) [reply recipes/pytorch-ignite/meta.yaml]: Hi @beenje 
Thank you for your suggestions, which I attempted to take into account. Please let me know if you have any further edits.

**Review by dopplershift — APPROVED** [review]

**Comment by dopplershift** [comment]
Ignoring Windows build failure due to missing pytorch since this is noarch.

**Pull request merged**